JumpscaleCore/clients/test/test_sonic_client.py

```diff
@@ -0,0 +1,318 @@
+import unittest
+from Jumpscale import j
+from random import randint
```

Comment:
Instead of random, use `j.data.idgenerator.generateRandomInt(100, 200)` .. you can also use it in the base class and then use it here